server/models/userModel.js

```diff
@@ -2,7 +2,7 @@ const { Pool } = require('pg');
 
 require('dotenv').config();
 
-const PG_URI = `postgres://${process.env.PG_Username}:${process.env.PG_Password}@castor.db.elephantsql.com/${process.env.PG_Username}`;
+const PG_URI = `postgres://${process.env.PG_Username}:${process.env.PG_Password}@kashin.db.elephantsql.com/${process.env.PG_Username}`;
```

Comment:
Nice you were able to change the db name. Should be in the .env file though. That is solved in our current working branch.